src/redfish/rest/v1.py

```diff
@@ -746,7 +746,9 @@ def _get_req_headers(self, headers=None):
         elif self.__authorization_key:
             headers['Authorization'] = self.__authorization_key
 
-        headers['Accept'] = '*/*'
+        if 'Accept' not in headers:
+            headers['Accept'] = '*/*'
+

```

Comment:
The header names are case-insensitive, so we can do something like this for the check in case the user-provided header is e.g. 'accept' or 'odata-version':
```
header_keys = set(k.lower() for k in headers)
if 'accept' not in headers:
    headers['Accept'] = '*/*'
```